Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde = { version = "1", features = ["derive"] }
 rand_core = "^0.6"
 getrandom = "0"
 sha2 = { version = "0.10", default-features = false, features = ["std", "oid"] }
-ed25519-compact = { version = "2", features = ["std", "traits"] }
+ed25519-compact = { version = "=2.0.6", features = ["std", "traits"] }
```

Comment:
Instead of pinning can you fix the issue so we don’t have to keep the ed crate in the past?

Reply:
> Instead of pinning can you fix the issue so we don’t have to keep the ed crate in the past?

Ok will try to do that